src/main/java/server/headRequestHandler.java

```diff
@@ -0,0 +1,13 @@
+package server;
+
+public class headRequestHandler implements Handler {
```

Comment:
**question** Is there a reason you went with lower case class names for the handlers?

Reply:
Good catch! Nope, just thinking of class instance names apparently!